app/src/main/java/io/left/meshim/controllers/RightMeshController.java

```diff
@@ -240,6 +247,8 @@ private void handleDataReceived(RightMeshEvent e) {
 
                 if (sender != null && user != null) {
                     Message message = new Message(sender, user, protoMessage.getMessage(), false);
+                   // message has been delivered
```

Comment:
This comment is probably redundant.

Actually, is the `Message` constructor used above only used when creating received messages? If so we can default  delivered to `true` in it and avoid this call.

Reply:
no it is used in sendTextMessage as well with different value for isMyMessage